test/test_docker_policy_bad_image.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+echo "Running './examples/bad_image.launch.py'"
+
+output=$(timeout -s INT 10s ./examples/bad_image.launch.py)
```

Comment:
This is neat - are we sure the `timeout` binary will always exist?

Reply:
```
apt-file search `which timeout`
coreutils: /usr/bin/timeout
```

Yes - it will

Reply:
timeout is part of GNU coreutils.